atomic_reactor/util.py

```diff
@@ -785,3 +787,71 @@ def get_config_from_registry(image, registry, digest, insecure=False,
     logger.debug('Image %s:%s has config:\n%s', context, tag, blob_config)
 
     return blob_config
+
+def df_parser(df_path, workflow=None, cache_content=False, env_replace=True, parent_env=None):
```

Comment:
@lcarva `DockerfileParser` is used through out `atomic-reactor` in places where workflow objects don't exist. Should I just leave those `DockerfileParser` and only use `df_parser` wrapper where necessary? My main motivation for going "all-in" was an attempt to not introduce ambiguity.